package.json

```diff
@@ -79,9 +83,16 @@
       "<rootDir>/tests/**/*.spec.ts"
     ]
   },
+  "lint-staged": {
+    "{,src/**/}*.{md,json,ts}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
   "scripts": {
-    "lint": "tslint --project tslint.json",
-    "lint:fix": "tslint --project tslint.json --fix",
+    "format": "prettier --write \"{,src/**/}*.{md,json,ts}\"",
```

Comment:
add also html file please

Reply:
I think we don't have any `.html` files in this library.

Reply:
Yes but in the future we can have some

Reply:
updated